app/models/annotation/nml/NmlParser.scala

```diff
@@ -62,7 +63,7 @@ object NmlParser extends LazyLogging with ProtoGeometryImplicits with ColorGener
         _ <- TreeValidator.validateTrees(treesSplit, treeGroupsAfterSplit, branchPoints, comments)
         additionalAxisProtos <- parseAdditionalAxes(parameters \ "additionalAxes")
         datasetName = overwritingDatasetName.getOrElse(parseDatasetName(parameters \ "experiment"))
-        organizationName = if (overwritingDatasetName.isDefined) None
+        organizationName = if (overwritingDatasetName.isDefined) overwritingOrganizationName
         else parseOrganizationName(parameters \ "experiment")
```

Comment:
Shouldn't the if-check check for `overwritingOrganizationName` being defined and not for `overwritingDatasetName`? Or is there some other place where it is ensured that in case `overwritingDatasetName` is defined, then `overwritingOrganizationName` is also defined?

And maybe the same syntax as in the line above can be used here as well, as imo the code is more readable this way. Or is there a semantic difference between my suggestion and your code?
```suggestion
        organizationName = overwritingOrganizationName.getOrElse(parseOrganizationName(parameters \ "experiment"))
```

Reply:
Yes, there is in fact a difference. I want the overwritingOrganizationName, even if it is None, if the overwritingDatasetName is defined.

This case will probably not happen, because the frontend sends both or neither. But from the backend point of view, the organizationName from the NML becomes invalid once the datasetName is overwritten (this was already the case before this PR). So in that moment I want to yield the None, so that it will later be filled by the disambiguation logic via dataset name, and not the organizationName from the NML.